skribenten-web/frontend/package.json

```diff
@@ -18,11 +18,14 @@
     "@navikt/aksel-icons": "5.9.2",
     "@navikt/ds-css": "5.9.2",
     "@navikt/ds-react": "5.9.2",
-    "@tanstack/react-location": "3.7.4",
     "@tanstack/react-query": "5.8.1",
+    "@tanstack/react-query-devtools": "5.7.2",
+    "@tanstack/react-router": "0.0.1-beta.212",
```

Comment:
Beta? 🤔

Reply:
Ja 😅 Er enig at noe som har beta i seg automatisk er litt sketch som avhengighet. Men det er fra samme team/fyr som react-query som har en utrolig bra track record. Dokumentasjonen er veldig gjennomført og det commites på daglig. Det er også en fork av den mest kjente SPA ruteren `react-router`. React-router hadde også vært et solid valg, men det denne her tilbyr er full TS støtte for alle ruter, parametere og søk. Det er hovedgrunnen til at jeg valgte den

Reply:
Ait